test/test_dpr.py

```diff
@@ -439,7 +442,91 @@ def test_dpr_save_load():
     dataset2, tensor_names, _ = loadedprocessor.dataset_from_dicts(dicts=[d], return_baskets=False)
     assert np.array_equal(dataset.tensors[0],dataset2.tensors[0])
 
+
+def test_dpr_training():
+    batch_size = 1
+    n_epochs = 1
+    distributed = False  # enable for multi GPU training via DDP
+    evaluate_every = 1
+    question_lang_model = "microsoft/MiniLM-L12-H384-uncased"
+    passage_lang_model = "microsoft/MiniLM-L12-H384-uncased"
+    do_lower_case = True
+    use_fast = True
+    similarity_function = "dot_product"
+
+
+
+    device, n_gpu = initialize_device_settings(use_cuda=False)
+
+    query_tokenizer = Tokenizer.load(pretrained_model_name_or_path=question_lang_model,
+                                     do_lower_case=do_lower_case, use_fast=use_fast)
+    passage_tokenizer = Tokenizer.load(pretrained_model_name_or_path=passage_lang_model,
+                                       do_lower_case=do_lower_case, use_fast=use_fast)
+    label_list = ["hard_negative", "positive"]
+
+    processor = TextSimilarityProcessor(query_tokenizer=query_tokenizer,
+                                        passage_tokenizer=passage_tokenizer,
+                                        max_seq_len_query=10,
+                                        max_seq_len_passage=10,
+                                        label_list=label_list,
+                                        metric="text_similarity_metric",
+                                        data_dir="samples/dpr/",
+                                        train_filename="sample.json",
+                                        dev_filename="sample.json",
+                                        test_filename=None,
+                                        embed_title=True,
+                                        num_hard_negatives=1,
+                                        dev_split=0,
+                                        max_samples=2)
+
+    data_silo = DataSilo(processor=processor, batch_size=batch_size, distributed=False)
+
+    question_language_model = LanguageModel.load(pretrained_model_name_or_path="bert-base-uncased",
```

Comment:
At the beginning, `question_lang_model` and `passage_lang_model` are defined as `"microsoft/MiniLM-L12-H384-uncased"`. Here, they are set to `"bert-base-uncased"`.

Reply:
good catch